### PR TITLE
Changes for building a ppc64le stemcell for Power8 Ubuntu LE

### PR DESCRIPTION
**Pull request by ipoddar-ibm**
Assumes that gccgo is installed under /usr/local/gccgo in the machine running the stemcell build commands.

### CODE REVIEW

**Comment by maximilien** [comment]
@ipoddar-ibm can you make sure to rebase this PR. I'd rather look at it after the rebase since that's something you'd need to do anyway.

Thanks.

File: stemcell_builder/stages/bosh_go_agent/apply.sh [hunk 1]
```diff
@@ -28,6 +28,11 @@ cd $assets_dir/go/src/github.com/cloudfoundry/bosh-agent
 bin/build
 
 mv out/bosh-agent $chroot/var/vcap/bosh/bin/
+if [ `uname -m` == "ppc64le" ]; then
+   # assume that gccgo is installed under /usr/local/gccgp
+   cp -rvH /usr/local/gccgo $chroot/var/vcap/bosh/
```
Comment (allomov) [inline stemcell_builder/stages/bosh_go_agent/apply.sh]: @ipoddar-ibm could you please tell, why did you need copying gccgo folder to `/var/vcap/bosh/` ?

**Comment by allomov** [comment]
I have rebased this changes to the latest BOSH stable tag (stable-3033). You can find diff with develop branch here: https://github.com/cloudfoundry/bosh/compare/develop...Altoros:power-3033-stemcell.

Still this changes are not completely ready for PR because of some dev artifacts. Also there are some things to be done:

1. I needed to remove rsyslog-mmjsonparse package, because it was absent in package sources.
1. I had problem installing system_ixgbevf.
1. I made changes that rely on this PRs: https://github.com/cloudfoundry/bosh-agent/pull/24 and https://github.com/cloudfoundry/bosh-davcli/pull/1.

**Comment by dpb587-pivotal** [comment]
Closing - feature set merged in https://github.com/cloudfoundry/bosh/pull/1027
